ui/infra-compare/InfraCompareTableControls.jsx

```diff
@@ -106,6 +114,23 @@ export default class CompareTableControls extends React.Component {
           updateFilter={this.updateFilter}
           updateFilterText={this.updateFilterText}
         />
+        <Label for="filterPercent">
+          Filter percentage: {this.state.filterPercent}%
```

Comment:
You already have `filterPercent` destructured on line 93, so you can use that variable here instead of `this.state.filterPercent`.

Reply:
DOne!!